### ISSUE TITLE
./teastart_minimal.sh: line 23: <pid> Killed 

### ISSUE THREAD

**Issue by Bluscream** [TeaSpeak/TeaSpeak#198]
## System Information
***Operating System***: Linux 4.4.0-127-generic #153-Ubuntu SMP Sat May 19 10:58:46 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
***TeaSpeak Version***: v1.1.35-beta-4 [Build: 1529242094]

## Issue
***Reason for reporting this issue***:
Server stops for no reason.

***Detailed Description***:
stdout:
```
[04:11:27] [INFO]   1 | [FILE] Missing client icon (4294967294).
[04:11:27] [INFO] GEN | Client Bluscream/XXXX= tried to join on a not existing channel. Name: /139
[04:11:27] [ERROR] Timer flush needs more than 16 milliseconds. Max allowed was 5 milliseconds.
[04:11:27] [ERROR] Timer statistics tick needs more than 48 milliseconds. Max allowed was 5 milliseconds.
[04:11:27] [ERROR] Timer db helper tick needs more than 40 milliseconds. Max allowed was 5 milliseconds.
[04:11:29] [ERROR] Timer license tick needs more than 8151 microseconds. Max allowed was 5000 microseconds.
[04:11:30] [ERROR]   1 | Found delay while ticking server. (Delay: 3197ms)
[04:11:30] [ERROR]   1 | Ticking of client XXXX:63035 (Bluscream) needs more that 2500 microseconds! (44401 microseconds)
[04:11:30] [INFO]   1 | [FILE] Missing client icon (4294967294).
[04:11:30] [ERROR]   1 | Server tick tooks to long (287)
[04:11:30] [ERROR] Timer flush needs more than 6219 microseconds. Max allowed was 5000 microseconds.
[04:11:31] [ERROR] Timer statistics tick needs more than 74 milliseconds. Max allowed was 5 milliseconds.
[04:11:32] [ERROR] Timer client join needs more than 4697 milliseconds. Max allowed was 10 milliseconds.
> ./teastart_minimal.sh: line 23: 11428 Killed                  ./TeaSpeakServer
root@XXX:/home/teaspeak# 
```
log_2018-06-18_04:06:31_general.log:
```
[04:11:27] [DEBUG]   1 | Client id: 1
[04:11:27] [INFO] GEN | Client Bluscream/XXXXX= tried to join on a not existing channel. Name: /139
[04:11:27] [ERROR] Timer flush needs more than 16 milliseconds. Max allowed was 5 milliseconds.
[04:11:27] [ERROR] Timer statistics tick needs more than 48 milliseconds. Max allowed was 5 milliseconds.
[04:11:27] [TRACE]   1 | [Property] Not saving property 'client_channel_group_inherited_channel_id', changed for 3 (New value: 4)
[04:11:27] [TRACE]   1 | [Property] Not saving property 'client_channel_group_id', changed for 3 (New value: 14)
[04:11:27] [ERROR] Timer db helper tick needs more than 40 milliseconds. Max allowed was 5 milliseconds.
[04:11:29] [ERROR] Timer license tick needs more than 8151 microseconds. Max allowed was 5000 microseconds.
[04:11:29] [TRACE]   1 | [Property] Not saving property 'client_talk_power', changed for 3 (New value: -2)
[04:11:30] [TRACE]   1 | [CLIENT] Updating manager icon from 0 to 4294967294
[04:11:30] [ERROR]   1 | Found delay while ticking server. (Delay: 3197ms)
[04:11:30] [DEBUG] GEN | Could not find requested file. Abs path: "./files/server_1/icons/icon_4294967294"|./files/server_1/icons/icon_4294967294. (path=./files/server_1/icons/icon_4294967294, parent=./files/server_1/icons)
[04:11:30] [ERROR]   1 | Ticking of client XXXX:63035 (Bluscream) needs more that 2500 microseconds! (44401 microseconds)
[04:11:30] [INFO]   1 | [FILE] Missing client icon (4294967294).
[04:11:30] [ERROR]   1 | Server tick tooks to long (287)
```

**Comment by WolverinDEV** [comment]
Hey,
Your system kills your server for some reason. 
You could find more detailed reason within the system log

**Comment by Bluscream** [comment]
Which one?
![grafik](https://user-images.githubusercontent.com/3318223/41519790-a880408a-72b9-11e8-86cf-c8889e7f878d.png)

**Comment by Bluscream** [comment]
Makes me think that this is related to #142

**Comment by WolverinDEV** [comment]
Should be fixed as well as #142
